<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>431</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_home">
     <widget class="QLabel" name="home_label_homepageTitle">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid Sans Fallback</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Dank Title</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_homepageButtons">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="home_pushButton_viewAllRestaurants">
         <property name="text">
          <string>View Nearby Restaurants</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="home_pushButton_planRegularFoodRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Plan a food run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="home_pushButton_planCustomFoodRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Plan a custom food run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_viewAllRestaurants">
     <widget class="QPushButton" name="viewAllRestaurants_pushButton_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAllRestaurants_pushButton_viewDetails">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>440</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>View Details</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="viewAllRestaurants_label_title">
         <property name="text">
          <string>View all Restaurants</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="viewAllRestaurants_tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_planRegularTrip">
     <widget class="QLabel" name="planRegularTrip_label_title">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan your trip!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planRegularTrip_pushButton_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="planRegularTrip_comboBox_startingLocation">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="planRegularTrip_comboBox_numberOfStops">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="planRegularTrip_label_promptLocations">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>How man locations would you like to visit?</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="planRegularTrip_label_promptStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Where would you like to start?</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="planRegularTrip_pushButton_go">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>87</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>GO!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_planCustomFoodRun">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>plan custom food run (temp label)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planCustomFoodRun_pushButton_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_viewDetails">
     <widget class="QPushButton" name="viewDetails_pushButton_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;insert restauraunt name&gt;'s Details</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdmin"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
